torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1378,6 +1378,44 @@ def _make_tensor_helper(shape, low=None, high=None):
 
     return samples
 
+def sample_inputs_rsub(op_info, device, dtype, requires_grad, variant='tensor'):
+    def _make_tensor_helper(shape, low=None, high=None):
+        return make_tensor(shape, device, dtype, low=low, high=high, requires_grad=requires_grad)
+
+    alpha = 2  # type: ignore
+    if dtype.is_complex:
+        alpha = 1 + 2j  # type: ignore
+    elif dtype.is_floating_point:
+        alpha = 0.1  # type: ignore
+
```

Comment:
consider also adding tests for these input variants:
real input, real other, complex alpha
real input, complex other, real alpha
complex input, real other, real alpha
complex input, real other, complex alpha
complex input, complex other, real alpha

Reply:
@kshitij12345 I think some of these cases are still not tested